packages/core/src/__integrationtests__/Did.spec.ts

```diff
@@ -49,7 +49,7 @@ let api: ApiPromise
 
 beforeAll(async () => {
   await initializeApi()
-  api = await BlockchainApiConnection.getConnectionOrConnect()
+  api = ConfigService.get('api')
```

Comment:
Should it be like in the other test?
```suggestion
  api = await initializeApi()
```

Reply:
why not. initially changed it in the other bc it was breaking for some reason